spec/services/hmrc/interface/submission_service_spec.rb

```diff
@@ -92,4 +92,44 @@ def error_response
       expect(call.keys).to match_array %i[id _links]
     end
   end
+
+  describe ".request_body" do
+    subject(:request_body) { interface.request_body }
+
+    context "when the owner is an applicant" do
+      let(:expected_data) do
```

Comment:
Can this be moved onto line 98 and so then it is only written once and reused in the second context block below?